server/src/main/scala/services/graphstages/FlightFilter.scala

```diff
@@ -0,0 +1,38 @@
+package services.graphstages
+
+import drt.shared.{AirportConfig, ApiFlightWithSplits, PortCode}
+import drt.shared.Terminals.{T2, T5, Terminal}
+import services.AirportToCountry
+
+case class FlightFilter(filters: List[ApiFlightWithSplits => Boolean]) {
+  def +(other: FlightFilter): FlightFilter = FlightFilter(filters ++ other.filters)
+  def apply(fws: ApiFlightWithSplits): Boolean = filters.forall(_(fws))
+}
+
+object FlightFilter {
+  def apply(filter: ApiFlightWithSplits => Boolean): FlightFilter = FlightFilter(List(filter))
+
+  case object ValidTerminalFilter {
```

Comment:
Do we need this as case object ? It can just be either def or case / object outside FlightFilter .

Reply:
Nothing wrong as such to use here just seen mostly used case object for define enum type

Reply:
I think you're right - it's the odd one out, but since all the others are just defined as simple functions I'll change this to be consistent :+1: